packages/docusaurus-theme-openapi-docs/src/theme/ApiDemoPanel/CodeTabs/_CodeTabs.scss

```diff
@@ -33,18 +49,13 @@
   }
 }
 
-.language-python {
```

Comment:
Went ahead and removed styling for setting `max-height` and `overflow` for `.language-` selectors and  only seting the `max-height` on `.openapi-demo__code-block code` to prevent triple vertical scrollbar effect. 

Before: 

https://user-images.githubusercontent.com/48506502/228043320-72a6116e-e553-401b-8c70-6fa0f28cac18.mov

After: 


https://user-images.githubusercontent.com/48506502/228043580-8fa53bbc-f4ba-4bbf-80aa-ca0372dbb193.mov